symfony/client/src/orangehrmPimPlugin/components/EditSupervisor.vue

```diff
@@ -0,0 +1,108 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-horizontal-padding orangehrm-top-padding">
+    <oxd-text tag="h6" class="orangehrm-main-title">Edit Supervisor</oxd-text>
+    <oxd-divider />
+    <oxd-form :loading="isLoading" @submitValid="onSave">
+      <oxd-form-row>
+        <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+          <oxd-grid-item>
+            <employee-dropdown v-model="reportTo.employee" />
```

Comment:
Please add `rules` and `required` props as it is a required field.

Reply:
done, removed this file,more generic component created for both supervisor and subordinate edit, added rules there